contracts/polygon/PolymorphChild.sol

```diff
@@ -47,4 +52,73 @@ contract PolymorphChild is IPolymorphChild, PolymorphWithGeneChanger {
         _mint(ownerAddress, tokenId);
         emit TokenMinted(tokenId, gene);
     }
-}
+
+    function setMaticWETHContract(address _maticWETHAddress) public onlyDAO {
```

Comment:
Can we also set the maticWETH contract in the constructor, so that we dont ask the DAO to go for a vote?

Reply:
Done.